scripts/build_experimental_archive.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Define build and output directories
+gitTarget=${1:-"master"}
+buildDir=${2:-"./build"}
+
+# Create build directory and navigate into it
+mkdir -p "${buildDir}" && cd "${buildDir}"
```

Comment:
You don't need the `&&` with `set -euxo pipefail`.

Also, I'd `rm -r ${buildDir}` before entering into it, otherwise `git clone` would fail the second time you run it. If you instead want to keep the `${buildDir}` and the cloned repository around, you need logic to avoid cloning the repository if it's already there. In such a case, you would need to pull the sources again.

Evidence:

```text
% git clone -v https://github.com/ooni/probe-cli.git
Cloning into 'probe-cli'...
POST git-upload-pack (175 bytes)
POST git-upload-pack (gzip 8652 to 4314 bytes)
remote: Enumerating objects: 29352, done.
remote: Counting objects: 100% (332/332), done.
remote: Compressing objects: 100% (220/220), done.
remote: Total 29352 (delta 141), reused 247 (delta 103), pack-reused 29020
Receiving objects: 100% (29352/29352), 29.08 MiB | 9.65 MiB/s, done.
Resolving deltas: 100% (20037/20037), done.

% git clone -v https://github.com/ooni/probe-cli.git
fatal: destination path 'probe-cli' already exists and is not an empty directory.
```